src/services/ray/src/ray/deployments/modeling/base.py

```diff
@@ -393,10 +406,14 @@ def stream_send(self, data: Any):
         Args:
             data (Any): The data to stream back to the client.
         """
-        
-        response = self.request.create_response(BackendResponseModel.JobStatus.STREAM, self.logger, data=data)
-        
-        SioProvider.emit("stream", data=(self.request.session_id,  response.pickle(), self.request.id))
+
+        response = self.request.create_response(
```

Comment:
Personal preference: If you are submitting a PR for something, please don't add in non-related functionality/linting.